main.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/ServiceWeaver/weaver"
+)
+
+var (
+	// The address on which the HTTP server serves locally.
+	localAddress = flag.String("address", "localhost:12347", "Local server address")
```

Comment:
s/7/5/ ?

Reply:
Clarified that I was trying to be clever by choosing a prime 12347 :)